composer.json

```diff
@@ -13,9 +13,10 @@
     },
     "require-dev": {
         "composer/installers": "~1.5",
+        "consolidation/robo": "~1.4",
+        "consolidation/annotated-command": "~2.12",
```

Comment:
These are dependencies of the task runner, we should rather increase the minimum version of the task runner and not hardcode those in here. Let's use `"openeuropa/task-runner": "~1.0.0beta6"` and see if it solves the problem.

Reply:
Thank you . It is fixed